siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -895,6 +906,9 @@ def _get_ps_common_propty_database():
             'type': 'float', 'value': 1.0, 'prec': 4, 'unit': 's',
             'lolo': 1e-2, 'low': 1e-2, 'lolim': 1e-2,
             'hilim': 4096, 'high': 4096, 'hihi': 4096},
+        # Period Sync Pulse
+        'PeriodSyncPulseCount-Mon': {
```

Comment:
the name PeriodSyncPulseCount (ans its corresponding PS variable) is not intuitive for what it meas. Discuss with @gabrielbrunheira and think of a better name.

Reply:
Maybe somethink like `ClockCountPerSyncPulse-Mon`?

Reply:
Discussing with Ana, we propose `CtrlCyclesBtwSyncCount-Mon`.

Reply:
Discussing with @xresende , we propose `BtwSyncCtrlCycleCount-Mon`, what do you think? The phrase would be "control cycle counts between sync pulses", so to keep Count at the end, we inverted the order

Reply:
I strongly go against using a PV name with `SyncCount`: I think most people will understand it to be the number of sync pulses, regardless of what comes as prefix. I suggest we keep `CtrlCycleCount`together. and if we count the PV name to end with `Count which makes sense since the PV is of a counter type, I suggest using `BtwSyncPulses`as a prefix or another one that we find more natural...., @gabrielbrunheira , can we try to give a priority to this so that we get to a consensus and @anacso17 can resume working on the test procedures? maybe by asking other people opinions, a well (poor guys...)

Reply:
@fernandohds564 , do you a say about the PV naming? (sorry! :smile: )

Reply:
maybe something like this: `NumCtrlCycBtwLastTrigs-Mon`?
I tried to avoid `Sync` and `Count`in this suggestion. Maybe it could also had the ending `...Last2Trigs-Mon`.

Reply:
It is not a counter in the sense of a PV counter (up to now implemented only as -Cmd, as far as I am remember). It is a counter for the power supply, this is what I meant. 
If we give up the idea of ending the name with a substring that refers to what the PV is (Count or Num)  that things get definitely easier. we can choose it. Is `Num`in our PV naming standard? Is it not `Nr` ? (not sure). I prefer `LastTrigs` over `Last2Trigs`. apart from these remarks, fernando's suggestion is ok for me.

Reply:
From wiki, we use `Nr`: https://wiki-sirius.lnls.br/mediawiki/index.php/Table:Naming_System_General

Reply:
So we have `NrCtrlCycBtwLastTrigs-Mon`?